CONTRIBUTING.md

```diff
@@ -38,6 +38,7 @@ below:
 * Benjamin Ayliffe (Met Office, UK)
 * Mark Baker (Met Office, UK)
 * Laurence Beard (Met Office, UK)
+Neil Crosswaite (Met Office, UK)
```

Comment:
This needs an * to make it a bullet point

Reply:
Fixed it